src/plugins/vis_type_timelion/public/components/bar_series/index.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { BarSeries, ScaleType } from '@elastic/charts';
+
+export function BarSeriesComponent({ data, index }: { data: any; index: number }) {
+  const bars = data.bars || {};
+  const styles = {
+    barSeriesStyle: {
+      rect: {
+        fill: data.color,
+        opacity: !bars.fill || bars.fill < 0 ? 1 : bars.fill,
```

Comment:
I think we should always:
- if fill is not available, then use 1 as default
- else use always `fill` but limit it to [0,1] range, like if the user specify -1 it goes to 0 or 10 goes to 1.

Reply:
fixed